common/src/main/java/bisq/common/util/Hex.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.common.util;
+
+import com.google.common.io.BaseEncoding;
+
+public class Hex {
+
+    public static byte[] decode(String hex) {
+        return BaseEncoding.base16().lowerCase().decode(hex.toLowerCase());
+    }
+
+    public static String encode(byte[] bytes) {
+        return BaseEncoding.base16().lowerCase().encode(bytes);
+    }
+}
```

Comment:
We have in the bisq.common.util.Utilities calls methods for encode and decode.

Reply:
Actually it would be more clean to use the class as you added here. But maybe in a second PR would be good to refactor the usage of the Utilities methods to have only one method used for hex. There might be also use of the HEX class from BitcoinJ (same impl).

Reply:
Yes, I am aware of Utilities and BitcoinJ versions of Hex being used in the code. It did cross my mind to standardize this but I decided it is too broad for this PR.

In this PR I focused on minimizing the diff and ensuring backwards compatibility. We need exactly the same behavior from replacements as original Bounce Castle versions.